scripts/zones/Jade_Sepulcher/mobs/Mocking_Colibri.lua

```diff
@@ -0,0 +1,34 @@
+-----------------------------------
+-- Area: Jade Sepulcher
+--   NM: Mocking Colibri
+-----------------------------------
+mixins = { require('scripts/mixins/families/colibri_mimic') }
+-----------------------------------
+local entity = {}
+
+entity.onMobInitialize = function(mob)
+end
+
+entity.onMobSpawn = function(mob)
+    mob:addImmunity(xi.immunity.SILENCE)
+    mob:addImmunity(xi.immunity.DARK_SLEEP)
+    mob:addImmunity(xi.immunity.LIGHT_SLEEP)
+end
+
+entity.onMobDisengage = function(mob)
+    -- Remove dots when the mob disengages such as when charming everyone
+    mob:delStatusEffect(xi.effect.POISON)
+    mob:delStatusEffect(xi.effect.BURN)
+    mob:delStatusEffect(xi.effect.FROST)
+    mob:delStatusEffect(xi.effect.CHOKE)
+    mob:delStatusEffect(xi.effect.RASP)
+    mob:delStatusEffect(xi.effect.SHOCK)
+    mob:delStatusEffect(xi.effect.DROWN)
+    mob:delStatusEffect(xi.effect.DIA)
+    mob:delStatusEffect(xi.effect.BIO)
```

Comment:
This is interesting. Is this something common?

Not for this PR, but I am wondering, if this is something common, if it should be globablized

Reply:
Not that I know of. I at first was using `delStatusEffectsByFlag(xi.effectFlags.ERASABLE)` but Melo tested and found that Slow would not be removed. I'm kind of surprised there isn't a DoT effect flag. I don't know if the flags are something we made up or if they are tied to retail in some way but if we can add some I'd say that'd be the way to do this. That's a whole other PR though.